components/ActionSheet/src/private/MDCActionSheetItemTableViewCell.m

```diff
@@ -105,6 +105,7 @@ - (void)commonMDCActionSheetItemViewInit {
   [self.contentView addSubview:_imageView];
   _imageView.translatesAutoresizingMaskIntoConstraints = NO;
   _imageView.alpha = ImageAlpha;
+  [_imageView setTintColor:[UIColor blackColor]];
```

Comment:
If you remove this and rely on recursive tintColor being applied will it still work?

Reply:
Done